src/Features/Core/Portable/Diagnostics/BuildToolId.cs

```diff
@@ -15,12 +17,12 @@ internal abstract class BuildToolId
 
         internal abstract class Base<T> : BuildToolId
```

Comment:
aside... naming a base class 'Base' is not helpful :)

Reply:
Agreed, I'll avoid refactoring/renaming in this PR though.